apps/main/next.config.js

```diff
@@ -27,7 +27,7 @@ const nextConfiguration = {
     unoptimized: true,
   },
   trailingSlash: true,
-  transpilePackages: ['onboard-helpers', 'ui'],
+  transpilePackages: ['curve-features', 'curve-ui-kit'],
```

Comment:
is this even needed at all?

Reply:
I guess yes, to `automatically transpile and bundle dependencies`

Reply:
shouldn't curve-lib be there, then? it's also in typescript, it wouldn't work without transpilation

Reply:
I guess yes, all the packages it can transpile